templates/sag-mft-config/template.yaml

```diff
@@ -0,0 +1,66 @@
+alias: sag-mft-config
+description: ActiveTransfer Server configuration
+
+environments:
+  default:
+    hostname: localhost       # machine name   
```

Comment:
parameter names should be prefixes with the product, e.g. mft.xxx

mft.port.number: 22
mft.port.hostname: 0.0.0.0
mft.port.protocol: FTP